### PR TITLE
[W5.11][F09-A4] Eldon Chung

### PR DESCRIPTION
**Pull request by eldon-chung**
Ready for review.

### CODE REVIEW

**Comment by md-azsa** [comment]
Hi Eldon!
Very practical feature added! This is very useful to find people when users aren't sure if the find command is case sensitive or not. Very good job on updating the user guide! Maybe you can update the MESSAGE_USAGE for your find command to suit the new insensitive case field 👍  
Other than that, great effort on the feature! 

Happy CNY! 😄

File: doc/UserGuide.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Format: `list`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is case insensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (k-l-a) [inline doc/UserGuide.md:60]: You should modify the examples as well to show that this is the case.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
```
Comment (k-l-a) [inline src/seedu/addressbook/commands/FindCommand.java:20]: Good effort on updating the documentation, but try to update the header comments as well.

**Review by k-l-a — COMMENTED** [review]
@Eldon-Chung Some comments added. Please close PR after reading.

**Pull request closed without merging**